ccf_transaction_processor/generate_signing_keys/gen_keys.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage ./gen_keys.sh <ip-address-of-CCF>
+
+# Copy KEYs for test
+cp ../CCF/build/workspace/pdo_tp_common/user1_privk.pem .
```

Comment:
The documentation mentions `user0` keys. Are these `user1` keys different? If so, are they also created at build time? And if so, it might be worth mentioning in the documentation that there are multiple TLS channel key pairs generated at build time.

Reply:
two additional pairs of user keys get generated as part of deployment  (this is if i use their default start script). I am sure there is a more convoluted way to use their scripts to only allocate one user. but given that we really do not care about publishing user keys for PDO, probably not worth the time. Sure, will add a note that there are two additional sets of user keys. Thank you!